mkdocs.yaml

```diff
@@ -0,0 +1,117 @@
+site_name: Deeporigin CLI and Python Client
+site_author: Deep Origin
+repo_url: https://github.com/formiclabs/deeporigin
+repo_name: formiclabs/deeporigin
+site_url: https://formiclabs.github.io/deeporigin/
```

Comment:
For now, we can use `https://formiclabs.github.io/deeporigin-client/`. (`-client`) is missing.

We can have Moush help with hosting to `deeporigin.io`.

Reply:
fixed